### ISSUE TITLE
Submitting a build to Coverity Scan

### ISSUE THREAD

**Issue by SebastianNiemann** [Mantella/Mantella#327]
After speaking to Coverity's senior staff engineer Mel, we got the advise to create a new project, as simply renaming a project thats ownership got transfered might not to the trick.

As this is already done, we still have to submit our first builds for analysis. I assume that this can only be done by either @OpusV or myself, as we may need to submit the first one using coverity build tools as told in the admin console, prior to submit builds via Travis CI again.

**Comment by SebastianNiemann** [comment]
This is also somehow urgent and should not be pending until the next Sprint (two weeks), as I am not sure if the registration gets auto-deleted, if we wont submit the first job within X days.

**Comment by SebastianNiemann** [comment]
I just submitted a build to Coverity Scan. Now we have to wait for the final application review, before we can start looking into the analysis.
